### PR TITLE
chore(security hub): improve securityhub_enabled check logic

### LINKED ISSUE #1839: [Bug]: cis_1.5_aws benchmark scan shows failures when security hub is enabled without standards
**What happened?**
When running prowler with the `--compliance cis_1.5_aws` option, failures are shown for the `securityhub_enabled` check when Security Hub is enabled, but does not have standards subscriptions enabled. This is reported as a failure of CIS AWS foundations v1.5 benchmark 4.16, even though the this benchmark only calls for the SecurityHub service (as opposed to the standards) to be enabled.

**How to reproduce it**
1. Confirm security hub is enabled in an AWS account / region: `aws securityhub describe-hub`
2. Confirm security hub standards are not enabled: `aws securityhub get-enabled-standards`
3. Run `prowler aws --compliance cis_1.5_aws`

**Expected behavior**
No failures reported if security hub is enabled without standards.

**Screenshots or Logs**
`aws securityhub describe-hub` output:
{
    "HubArn": "[redacted],
    "SubscribedAt": "[redacted]",
    "AutoEnableControls": [redacted]
}

`aws securityhub get-enabled-standards` output:
{
    "StandardsSubscriptions": []
}

Prowler output: 
https://github.com/prowler-cloud/prowler/issues/1838

**From where are you running Prowler?**
Please, complete the following information:
 - Resource: workstation
 - OS: MacOS
 - Prowler Version [`./prowler --version`]: 3.1.1
 - Python version [`python --version`]: 3.10
 - Pip version [`pip --version`]: 22.3.1
 - Installation method (Are you running it from pip package or cloning the github repo?): pip package
 - Others:

**Additional context**
Security team who opt to use Prowler for CIS compliance will often want to disable SecurityHub standards in order to reduce noise in security hub. Also, SecurityHub standards currently do not include the CIS AWS foundations benchmark version 1.5

### PR DESCRIPTION
**Pull request by MrCloudSec**
### Description

Regarding https://github.com/prowler-cloud/prowler/issues/1839, improve `securityhub_enabled` check logic so if an user has Security Hub enabled but with integrations is also a PASS.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prowler/providers/aws/services/securityhub/securityhub_service.py [hunk 1]
```diff
@@ -32,67 +34,73 @@ def __threading_call__(self, call):
...
-                # SecurityHub is active so get HubArn
-                hub_arn = regional_client.describe_hub()["HubArn"]
                 if not self.audit_resources or (
                     is_resource_filtered(hub_arn, self.audit_resources)
                 ):
                     hub_id = hub_arn.split("/")[1]
+                    get_enabled_standards_paginator = regional_client.get_paginator(
+                        "get_enabled_standards"
+                    )
+                    standards = ""
```
Comment (jfagoagas) [inline prowler/providers/aws/services/securityhub/securityhub_service.py:61]: Why a string here?

File: prowler/providers/aws/services/securityhub/securityhub_service.py [hunk 2]
```diff
@@ -32,67 +34,73 @@ def __threading_call__(self, call):
...
-            else:
-                logger.error(
-                    f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
-                )
+            logger.error(
+                f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+            )
 
 
 @dataclass
```
Comment (jfagoagas) [inline prowler/providers/aws/services/securityhub/securityhub_service.py]: Can you use the Pydantic `BaseModel` here? We have to migrate all the classes we have.

**Review by jfagoagas — CHANGES_REQUESTED** [review]

File: prowler/providers/aws/services/securityhub/securityhub_service.py (hunk 2 above)
Comment (MrCloudSec) [reply prowler/providers/aws/services/securityhub/securityhub_service.py]: Done

File: prowler/providers/aws/services/securityhub/securityhub_service.py (hunk 1 above)
Comment (MrCloudSec) [reply prowler/providers/aws/services/securityhub/securityhub_service.py:61]: To parse the list and show it in the result extended.

**Review by jfagoagas — APPROVED** [review]

**Pull request merged**